src/startupscriptgenerator/src/python/main.go

```diff
@@ -107,7 +107,7 @@ func main() {
 			fmt.Sprintf("echo %s/lib > /etc/ld.so.conf.d/python.conf\n", pythonInstallationRoot))
 		scriptBuilder.WriteString("ldconfig\n")
 
-		if strings.HasPrefix(buildManifest.PythonVersion, "3.") {
+		if strings.HasPrefix(buildManifest.PythonVersion, "3.") && !strings.HasPrefix(buildManifest.PythonVersion, "3.10") {
```

Comment:
Will this be an issue if python 3 has patch version?

Reply:
These symlinks are for python version <= 3.9, for python >= 3.10 and all its patch versions symlinks are set up from source and we skip this step otherwise it'll throw errors.

Reply:
It's probably not worth delaying this PR for our partners for this, but it would be worth adding a comment explaining this in the code next time we have a chance. Looking at the code, it's hard to know why we're excluding this.